samples/amplitude-sample/src/app/sample.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { IConfigCatClient } from 'configcat-common/lib/ConfigCatClient';
+import { User } from 'configcat-common/lib/RolloutEvaluator';
+import amplitude from 'amplitude-js';
+
+@Component({
+    selector: 'app-sample',
+    templateUrl: 'sample.component.html',
+    styleUrls: ['./sample.component.scss']
+})
+
+export class SampleComponent implements OnInit {
+    constructor() { }
+
+    public isGreenButtonEnabled: boolean = undefined;
+    @Input() configCatClient: IConfigCatClient;
+
+    ngOnInit() {
+        const userObject = new User(amplitude.getInstance().options.deviceId);
```

Comment:
comment kene: mi ez a deviceID es miert rakjuk bele a user-be?
+ link a megfelelo helyre a CC doksiban.